### DISCUSSION TITLE
RE: Status of this project

### DISCUSSION THREAD

**Discussion by rotexdegba** [Josantonius/PHP-Session#23] [General]
Hi There,

I am looking to use this library in a web application that would be running for like the next ten years. Is this project still being maintained?

Thanks

**Comment by josantonius** [comment]
Hi!

Yes, I'm still maintaining it. I just updated it to include tests for PHP version 8.3.

I don't know if I'll be maintaining it for another 10 years, though! :sweat_smile:

_I'm moving this to [Discussions](https://github.com/josantonius/php-session/discussions), where it should be._

**Reply by rotexdegba** [reply]
Thanks for the feedback. In case you decide you can't continue maintaining this project, please give the community a heads up so either a fork could be created or you could hand over control of the repo to developers you trust.

Cheers!
